app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -130,6 +132,11 @@ function useUspsLocations() {
         address: bestMatchedAddress.address,
       });
     } else if (addressCandidates) {
+      trackEvent('IdV: in person proofing location search submitted', {
```

Comment:
adding analytics event to reflect no arcgis results here instead of in controller now fires event a single time. But frontendLogger appends "frontend" to the input string. Creating a new search event in the js analytics file using trackEvent still appends "frontend". Open to suggestions as to how to avoid that.

Reply:
added analytics event to eventMap in frontend_logger to address this